library/src/main/java/com/schibsted/spain/barista/assertion/BaristaVisibilityAssertions.kt

```diff
@@ -70,4 +72,23 @@ object BaristaVisibilityAssertions{
         onView(withText(text)).check(matches(not(isDisplayed())))
     }
 
+    @JvmStatic
+    fun assertContains(text: String) {
+        assertDisplayed(withText(CoreMatchers.containsString(text)))
```

Comment:
Really small mimimimi here: You're using the class prefix here but not in the methods below. Probably you can remove it.

Reply:
Update!